apps/condo/domains/user/integration/appleid/AppleIdIdentityIntegration.js

```diff
@@ -0,0 +1,154 @@
+// eslint-disable-next-line import/order
+const axios = require('axios').default
+
+// eslint-disable-next-line import/order
+const dayjs = require('dayjs')
+const jose = require('jose')
+const jwtDecode = require('jwt-decode')
+const { isNil } = require('lodash')
+
+const conf = require('@open-condo/config')
+
+const { normalizeEmail } = require('@condo/domains/common/utils/mail')
+
+// get apple id configuration params
+const APPLE_ID_CONFIG = process.env.APPLE_ID_CONFIG ? JSON.parse(process.env.APPLE_ID_CONFIG) : {}
+const {
+    authorizeUrl,
+    tokenUrl,
+    scope,
+    clientId,
+    redirectUri,
+    keyId,
+    teamId,
+    secretKey,
+} = APPLE_ID_CONFIG
+const callbackPath = '/api/apple_id/auth/callback'
+const callbackUri = redirectUri || `${conf.SERVER_URL}${callbackPath}`
+const axiosTimeout = 10000
```

Comment:
Constants should be written in screaming snake case
```suggestion
const AXIOS_TIMEOUT = 10000
```

Reply:
renamed